contrib/scripts/cubrid.csh

```diff
@@ -1,5 +1,3 @@
-
-setenv CUBRID /opt/cubrid
 setenv CUBRID_DATABASES $CUBRID/databases
```

Comment:
please add a license comment like cubrid.sh.

Reply:
Thank you for your comment, License text is added.